WooCommerce/Classes/ViewRelated/Dashboard/DashboardView.swift

```diff
@@ -178,9 +178,47 @@ private extension DashboardView {
                     }
                 }
             }
+
+            if canShowShareCard {
+                shareStoreCard
+            }
         }
     }
 
+    var shareStoreCard: some View {
+        VStack {
+            Image(uiImage: .blazeSuccessImage)
+                .padding(Layout.imagePadding)
+
+            Text(Localization.ShareStoreCard.title)
+                .headlineStyle()
+                .multilineTextAlignment(.center)
+
+            Text(Localization.ShareStoreCard.subtitle)
+                .bodyStyle()
+                .multilineTextAlignment(.center)
+                .padding(.horizontal, Layout.elementPadding)
+                .padding(.top, Layout.textPadding)
+
+            if let url = viewModel.siteURLToShare {
```

Comment:
I was thinking about this. Without the button, there is still a text that says "Use email or social media to spread the word about your store". This is just a suggestion but I feel might still be useful, so I left it in even without the button.  Another reason is also because I don't want the screen to be empty if we don't show this card due to missing URL.

Now that I think of it, what if we keep the button but keep the message generic? "Come visit my store!", for example. Then, a merchant can fill in the URL themselves as they see fit in the email/social media app.

Reply:
I think `ShareLink` requires an URL right? But your explanation makes sense, please feel free to disregard my suggestion (esp. after the explanation about .org stores with launch status).

Reply:
> I think ShareLink requires an URL right?

Oh yeah, good point here! I'll keep things this way for now.